src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -1,7 +1,24 @@
-import { darken } from 'polished'
-import { sizes, colors, transitions } from '@/shared/theme'
+import { colors, sizes, transitions } from '@/shared/theme'
 import { css } from '@emotion/react'
 import styled from '@emotion/styled/'
+import { darken } from 'polished'
+import Text from '../Text'
+
+type CheckboxLabelProps = {
+  disabled?: boolean
+}
+
+export const CheckboxLabel = styled.label<CheckboxLabelProps>`
+  padding: ${sizes(2)} 0;
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
+  align-items: center;
+  display: flex;
```

Comment:
```suggestion
  display: inline-flex;
```

Only the area with text should be clickable

Reply:
Once you make that change, the stories will be slightly broken so may need to put them in some container or something